svc-api/go.mod

```diff
@@ -3,9 +3,18 @@ module github.com/ODIM-Project/ODIM/svc-api
 go 1.13
 
 require (
-	github.com/ODIM-Project/ODIM/lib-persistence-manager v0.0.0-20201021053518-88345d4a9988 // indirect
-	github.com/ODIM-Project/ODIM/lib-utilities v0.0.0-20201015111324-83393fdedb82
+	github.com/Joker/hpp v1.0.0 // indirect
+	github.com/ODIM-Project/ODIM/lib-persistence-manager v0.0.0-20201201072448-9772421f1b55 // indirect
+	github.com/ODIM-Project/ODIM/lib-utilities v0.0.0-20201201072448-9772421f1b55
+	github.com/k0kubun/colorstring v0.0.0-20150214042306-9440f1994b88 // indirect
```

Comment:
why this package is added

Reply:
This package got added after the go mod tidy